### PR TITLE
Added dispute api

### PR DESCRIPTION
**Pull request by ankitdas13**
Api doc reference

[Fetch All Disputes](https://razorpay.com/docs/api/disputes/fetch-all)
[Fetch a Dispute](https://razorpay.com/docs/api/disputes/fetch)
[Accept a Dispute](https://razorpay.com/docs/api/disputes/accept)
[Contest a Dispute](https://razorpay.com/docs/api/disputes/contest)

### CODE REVIEW

File: documents/dispute.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+```
+
+**Parameters:**
+
+| Name  | Type      | Description                                      |
+|-------|-----------|--------------------------------------------------|
+| disputeId*  | string | The unique identifier of the dispute.  |
+| amount  | integer | The amount being contested. If the contest amount is not mentioned, we will assume it to be a full dispute contest.  |
+| summary  | string | The explanation provided by you for contesting the dispute. It can have a maximum length of 1000 characters. |
+| shipping_proof  | array | The explanation provided by you for contesting the dispute. It can have a maximum length of 1000 characters. |
```
Comment (razorpay-sanjib) [inline documents/dispute.md:178]: Please check the description of the shipping_proof

File: razorpay/__init__.py [hunk 2]
```diff
@@ -53,5 +54,6 @@
         'Stakeholder',
         'Product',
         'Iin',
-        'Webhook'
+        'Webhook',
+        'Dispute'
```
Comment (razorpay-sanjib) [inline razorpay/__init__.py:58]: add `,` at the end

File: razorpay/resources/__init__.py [hunk 3]
```diff
@@ -45,5 +46,6 @@
     'Stakeholder',
     'Product',
     'Iin',
-    'Webhook'
+    'Webhook',
+    'Dispute'
```
Comment (razorpay-sanjib) [inline razorpay/resources/__init__.py:50]: add `,` at the end

**Review by razorpay-sanjib — CHANGES_REQUESTED** [review]

File: documents/dispute.md (hunk 1 above)
Comment (ankitdas13) [reply documents/dispute.md]: made correction for `shipping_proof `

File: razorpay/__init__.py (hunk 2 above)
Comment (ankitdas13) [reply razorpay/__init__.py]: added

File: razorpay/resources/__init__.py (hunk 3 above)
Comment (ankitdas13) [reply razorpay/resources/__init__.py]: added

**Review by razorpay-sanjib — APPROVED** [review]
lgtm

**Pull request merged**